pkg/storegateway/bucket_test.go

```diff
@@ -2510,13 +2515,13 @@ func BenchmarkBucketBlock_readChunkRange(b *testing.B) {
 	}
 }
 
-func BenchmarkBlockSeries(b *testing.B) {
+func BenchmarkBlockSeriesSkippingChunks(b *testing.B) {
```

Comment:
Curious: what's the motivation behind changing the semantics of this test? Was is redundant / useless ?

Reply:
I think I see the reason in bucket.go, where `blockSeriesSkippingChunks` is the new function